### DISCUSSION TITLE
Banca dati Servizio Contratti Pubblici

### DISCUSSION THREAD

**Discussion by tbuon** [ondata/italian-public-sector-pnrr-data-guide#15] [Proposte]
### Testo

La banca dati sopra per come descritta contiene i dati sui bandi/avvisi, ma questi non sono i bandi/avvisi utili a selezionare un progetto per il PNRR ma sono quelli idonei ad indire una gara pubblica all'interno di un progetto già selezionato. Quindi, è cosa ben diversa.

**Comment by aborruso** [comment]
Ciao @tbuon e grazie ancora.

In quella pagina non abbiamo specificato che sono strettamente legati al PNRR. Ma la tua nota è giusta, e l'abbiamo aggiunta nella pagina.

![image](https://user-images.githubusercontent.com/30607/236928113-34d95229-051e-4fe0-b025-02d173647685.png)


Era presente già una nota in fondo https://pnrr.datibenecomune.it/fonti/scp/#i-dati-correlati-al-pnrr

**Comment by girolamodaschio** [comment]
Aggiungo una nota su tema Banca Dati Servizio Contratti Pubblici.

Nei giorni scorsi ho notato differenze nella presenza dei dati tra le api del sito [servizio contratti pubblici]( https://www.serviziocontrattipubblici.it/ocds-ms/swagger-ui.html) e la query al datastore del MIT, segnalata su [SCP: accesso ai dati](https://pnrr.datibenecomune.it/fonti/scp/informazioni-scp.html).

Una cig presente nel sito non era presente nel mit.
Vorrei allegare degli esempi, ma in questo momento il sito del MIT è in down.
![image](https://github.com/ondata/italian-public-sector-pnrr-data-guide/assets/58935223/73371c79-15ae-4680-bb65-4a642dea4a09)

Proposta 1:
Aggiungere alla pagina di Datibenecomune il riferimento anche al [sito servizio contratti pubblici](https://www.serviziocontrattipubblici.it/SPInApp/it/open_data.page).

Proposta 2:
Aggiungere alla pagina di [Datibenecomune](https://pnrr.datibenecomune.it/fonti/scp/informazioni-scp.html) un healtcheck sulle API.

**Reply by aborruso** [reply]
@girolamodaschio perché non fai una PR per ognuna delle proposte?

Il sito è basato su [quarto](https://quarto.org/), e i file (laddove non ci sono calcoli) sono in markdown. Puoi anche creare semplicemente due file md nuovi, che poi magari definiamo insieme dove mettere.

Che ne pensi?
